doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -67,6 +67,7 @@ nRF Desktop
   * Possibility to ask for bootloader variant using config channel.
   * Added Kconfig options that allow erasing dongle bond on the gaming mouse using buttons or config channel.
   * Added two states to enable erasing dongle peer: ``STATE_DONGLE_ERASE_PEER`` and ``STATE_DONGLE_ERASE_ADV``.
+  * Added configuration for ``nrf52833dk_nrf52820`` to make the ``nrf52820dongle`` publicly available.
```

Comment:
`... to allow testing nRF Desktop dongle on nrf52820 SoC.`
nrf52820 dongle is still not publicly available I think.